File: OnboardApps/python/DemoApp1/demoapp1.py
#!/usr/bin/env python
import time
from flyt_python import api
drone = api.navigation(timeout=120000) # instance of flyt droneigation class

#at least 3sec sleep time for the drone interface to initialize properly
time.sleep(3)

print 'taking off'
drone.take_off(5.0)

print ' going along the setpoints'
drone.position_set(5,0,0,relative=True)
drone.position_set(0,5,0,relative=True)
drone.position_set(-5,0,0,relative=True)
drone.position_set(0,-5,0,relative=True)

print 'Landing'
drone.land(async=False)

#shutdown the instance
drone.disconnect()
 

File: CPP-Python-ROS-Apps/cpp/DemoApp1/demoapp1.cpp
#include <core_script_bridge/navigation_bridge.h>
#include <iostream>

Navigation nav;
int main(int argc, char *argv[])
{
    nav.take_off(3.0);						//Taking Off
    nav.position_set(5,0,-3);				//Sending Position Setpoints
    nav.position_set(5,5,-3);
    nav.position_set(0,5,-3);
    nav.position_set(0,0,-3);
    nav.land(false);									//Landing
}
